nemo_text_processing/text_normalization/en/taggers/cardinal.py

```diff
@@ -44,6 +44,7 @@ def __init__(self, deterministic: bool = True, lm: bool = False):
         self.deterministic = deterministic
         # TODO replace to have "oh" as a default for "0"
         graph = pynini.Far(get_abs_path("data/number/cardinal_number_name.far")).get_fst()
+        graph_au = pynini.Far(get_abs_path("data/number/cardinal_number_name_au.far")).get_fst()
```

Comment:
the far file is not added

Reply:
It's been added. https://github.com/NVIDIA/NeMo-text-processing/blob/5629512439662a1b7dbb359315c877ad63c6f363/nemo_text_processing/text_normalization/en/data/number/cardinal_number_name_au.far